src/api/content-rest-api/api/categories.api.ts

```diff
@@ -372,9 +372,9 @@ export class CategoriesApi extends BaseApi {
     parameter, then the fields specified in the **include**
     parameter are returned in addition to those specified in the **fields** parameter.
 
-    * @return Promise<CategoryEntry>
+    * @return Promise<CategoryPaging | CategoryEntry>
     */
-    createSubcategory(categoryId: string, categoryBodyCreate: CategoryBody[], opts?: any): Promise<CategoryEntry> {
+    createSubcategories(categoryId: string, categoryBodyCreate: CategoryBody[], opts?: any): Promise<CategoryPaging | CategoryEntry> {
```

Comment:
Here we discuss that you will change also type I guess? 
https://github.com/Alfresco/alfresco-ng2-components/pull/8382#discussion_r1138282735

Can you remove CategoryEntry from return type as it does not seem needed? You pass list of categories so it will never give you single CategoryEntry

Reply:
It gives actually CategoryEntry in case if you pass array of 1 category